src/App.vue

```diff
@@ -3,42 +3,43 @@
     <!-- Dialogs -->
     <dashboard-unavailable-dialog
       :dialog="dashboardUnavailableDialog"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <business-auth-error-dialog
       :dialog="businessAuthErrorDialog"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <name-request-auth-error-dialog
       :dialog="nameRequestAuthErrorDialog"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <name-request-invalid-dialog
       :dialog="nameRequestInvalidDialog"
       :type="nameRequestInvalidType"
-      @exit="onClickExit"
-      @retry="onClickRetry"
+      @exit="onClickExit()"
+      @retry="onClickRetry()"
       attach="#app"
     />
 
     <!-- Initial Page Load Transition -->
-    <transition name="fade">
+    <v-fade-transition>
```

Comment:
The previous transition didn't work. This one does (fade prevents white flash when this block is displayed).